SonarQube.Bootstrapper/Program.cs

```diff
@@ -29,6 +29,21 @@ public static int Main(string[] args)
             var logger = new ConsoleLogger();
             Utilities.LogAssemblyVersion(logger, typeof(Program).Assembly, Resources.AssemblyDescription);
             IBootstrapperSettings settings;
+
+            if (ArgumentProcessor.IsHelp(args))
+            {
+                logger.LogInfo("");
+                logger.LogInfo("Usage: ");
+                logger.LogInfo("");
+                logger.LogInfo("  {0} [begin|end] /key:project_key [/name:project_name] [/version:project_version] [/d:sonar.key=value] ", System.AppDomain.CurrentDomain.FriendlyName);
+                logger.LogInfo("");
+                logger.LogInfo("    When executing the begin phase, at least the project key must be defined.");
+                logger.LogInfo("    Other properties can dynamically be defined with '/d:'. For example, '/d:sonar.verbose=true'.");
+                logger.LogInfo("    In the end phase no parameter should be passed.");
```

Comment:
We can also use the following, `/k` instead of `/key`, `/n` instead of `/name` and `/v:` instead of `/version`